examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -93,7 +93,8 @@ class TravelDestinationItem extends StatelessWidget {
                       child: FittedBox(
                         fit: BoxFit.scaleDown,
                         alignment: Alignment.centerLeft,
-                        child: Text(destination.title,
+                        child: Text(
```

Comment:
Text's positional parameter is inconsistently formatted in the flutter sources in general. I think handling this consistently, as dartfmt has done here, is an improvement.